src/iOS.Core/Controllers/BaseLockPasswordViewController.cs

```diff
@@ -60,6 +60,8 @@ public BaseLockPasswordViewController(IntPtr handle)
 
         public string BiometricIntegritySourceKey { get; set; }
 
+        public bool HasLoginMethod => _hasMasterPassword || _biometricEnabled || _pinEnabled;
```

Comment:
maybe rename this to `HasLoginOrUnlockMethod` to cover bio and pin?

Reply:
Agreed